olm_deploy/scripts/registry-init.sh

```diff
@@ -19,6 +19,9 @@ sed -i "s,quay.io/openshift/origin-elasticsearch-proxy:latest,${IMAGE_ELASTICSEA
 sed -i "s,quay.io/openshift/origin-oauth-proxy:latest,${IMAGE_OAUTH_PROXY}," /manifests/*/*clusterserviceversion.yaml
 sed -i "s,quay.io/openshift/origin-logging-kibana6:latest,${IMAGE_LOGGING_KIBANA6}," /manifests/*/*clusterserviceversion.yaml
 
+# update the manifest to pull always the operator image
+sed -i 's,imagePullPolicy:\ IfNotPresent,imagePullPolicy:\ Always,' /manifests/*/*clusterserviceversion.yaml
```

Comment:
@bparees this existed before the change.  any reason not to merge?  this workflow is intended to satisfy the dev experience correct?

Reply:
Correct this is to satisfy dev experience. However, we can make this conditional to `IMAGE_FORMAT` not in set.

Reply:
yeah please make it conditional so that when we run in CI we're running with a CSV that, as closely as possible, matches what we're going to ship (and what ART is going to build).